framework/source/class/qx/ui/form/core/VirtualDropDownList.js

```diff
@@ -407,28 +458,43 @@ qx.Class.define("qx.ui.form.core.VirtualDropDownList",
 
     /**
      * Adjust the drop-down to the available width and height, by calling
-     * {@link #__adjustWidth} and {@link #__adjustHeight}.
+     * {@link #_adjustWidth} and {@link #_adjustHeight}.
      */
-    __adjustSize : function()
+    _adjustSize : function()
     {
```

Comment:
Plese explain why you changes `__adjustSize`, `__adjustHeight` and `adjustWidth` to protected.

Reply:
So that you can adjust the size handling in a subclass; I thought it might be useful to someone, though I'm not taking advantage of this myself.

Reply:
OK. Its just that we want to have the PRs only contain changes related to the subject/enhancement or the bug fix. Let's see what the other say.

Reply:
I see, thank you for your patience in explaining it to me.

Reply:
I'm ok with the change to protected, if the ultimate commit message with PR merging mentions it.

Reply:
I don't mind either.